src/icalendar/prop/__init__.py

```diff
@@ -1736,6 +2234,44 @@ def from_ical(ical):
         except Exception as e:
             raise ValueError(f"Expected time, got: {ical}") from e
 
+    @classmethod
+    def examples(cls) -> list[vTime]:
+        """Examples of vTime"""
+        return [cls(time(12, 30))]
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jcal represenation of this property according to :rfc:`7265`."""
+        value = self.dt.strftime("%H:%M:%S")
+        if self.params.is_utc():
+            value += "Z"
+        return [name, self.params.to_jcal(exclude_utc=True), self.VALUE.lower(), value]
+
+    @classmethod
+    def parse_jcal_value(cls, jcal: str) -> timedelta:
+        """Parse a jcal string to a datetime.time.
+
+        Raises:
+            JCalParsingError
+        """
+        match = TIME_JCAL_REGEX.match(jcal)
+        if match is None:
+            raise JCalParsingError(f"Cannot parse {jcal!r} to a datetime.time.")
+        hour = int(match.group("hour"))
+        minute = int(match.group("minute"))
+        second = int(match.group("second") or 0)
```

Comment:
`or 0` is not required

Reply:
6a3bb7a you are right ... The seconds are required by the standard.